terraform/azure/infrastructure.tf

```diff
@@ -313,3 +313,34 @@ resource "azurerm_network_security_group" "mysecgroup" {
     workspace = local.deployment_name
   }
 }
+
+# IBSM
+data "azurerm_virtual_network" "ibsm_vnet" {
+  count               = var.enable_az_vnet_peering ? 1 : 0
+  name                = var.ibsm_vnet_name
+  resource_group_name = var.ibsm_rg
+}
+
+# Peering from the IBSM vnet to deployment vnet
+resource "azurerm_virtual_network_peering" "ibsm_to_target" {
+  count = var.enable_az_vnet_peering ? 1 : 0
+
+  name                      = "${var.ibsm_vnet_name}-to-${local.vnet_name}"
```

Comment:
Let's try this one to match https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/09e4331173c538f956294da7d788098205da7911/lib/qesapdeployment.pm#L2066

```
my $peering_name = "$source_vnet-$target_vnet";
```